timer/src/main/java/com/toggl/timer/startedit/ui/StartEditDialogFragment.kt

```diff
@@ -183,6 +184,7 @@ class StartEditDialogFragment : BottomSheetDialogFragment() {
             }
 
             done_action.setOnClickListener {
+                requireContext().performClickHapticFeedback()
```

Comment:
I don't think we need to `requireContext` for an action that's so non-critical like a little vibration
```suggestion
                context?.performClickHapticFeedback()
```